test/e2e/egress_firewall.go

```diff
@@ -290,6 +305,37 @@ spec:
 
 		}
 
+		ginkgo.By("Selecting additional IP addresses for serverNode (networking routing to secondaryIP address on other nodes is harder to achieve)")
```

Comment:
I dont understand what "networking routing to secondaryIP address on other nodes is harder to achieve" is in relation to. Can you elaborate on what you mean by this?

Reply:
good point, :) so I add 172.18.1.13 to ovn-control plane, I add 172.18.1.14 to ovn-worker and 172.18.1.15 to ovn-worker2, these are just IPs and we don't really add routes to the host for routing this. So if src pod.Node == dst Node, same node case it works in other cases we need to add routes to tell the node how to reach the 172.18 subnet which is a pain.

Reply:
so i basically only did pod on nodeA => secondaryIPs on nodeA test
I didn't do pod on nodeA => secondaryIPs on nodeB and nodeC test
If you insist I can change this to try to include that